### ISSUE TITLE
Implementation on CUB dataset

### ISSUE THREAD

**Issue by ShenghaiRong** [zxhuang1698/interpretability-by-parts#4]
**Thanks for your great work.**  When I transplanted the code to the CUB task, I just got the 86.4% accuracy. :( Specifically,  the linear classifier on celeb task is replaced with a single linear(2048, 200) coresponding with a cross entropy loss. And the backbone is Resnet101. The experiments also followed the implementation details in the paper. I wonder if there are still some differences between the reimplementation and the source code.
 And would you please release the code or reveal more training details on CUB dataset?

**Comment by zxhuang1698** [comment]
Hi, thanks for your interest in our work. The code has been updated.
